release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -120,6 +120,16 @@ For {product-title} {product-version}, you can quickly install a cluster on Amaz
 
 For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Intall a cluster quickly in AWS Local Zones].
 
+[id="ocp-4-14-install-update-cco-manual-enhancements"]
+==== Simplified installation and update experience for clusters with manually maintained cloud credentials
+
+This release includes changes that improve the experience of installing and updating clusters that use the Cloud Credential Operator (CCO) in manual mode for cloud provider authentication. The following parameters for the `oc adm release extract` command simplify the manual configuration of cloud credentials:
+
+`--included`:: Use this parameter to extract only the manifests that your specific cluster configuration needs. If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` custom resources for any disabled components before installing or updating a cluster. This parameter also makes the CCO utility (`ccoctl`) `--enable-tech-preview` parameter unnecessary.
```

Comment:
Most likely for 4.14 GA:
```suggestion
`--included`:: Use this parameter to extract only the manifests that your specific cluster configuration needs. If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` custom resources for any disabled components before installing or updating a cluster.
```

Reply:
We may want to work towards removing `--enable-tech-preview` in the future.  Possible migration plan:

* Release note in 4.14 that `--enable-tech-preview` is expected go away, possibly explaining the later steps in the plan, or maybe not to avoid committing before they're all that pinned down?
* 4.15 flips the default to "process all requests unless `--enable-tech-preview=false`" and logs to stderr if you use `--enable-tech-preview` (openshift/cloud-credential-operator#601) and drops doc references to `--enable-tech-preview`, other than saying it's deprecated.
* 4.16 (or could be later 4.y if we want) drops the `ccoctl` filtering and the flag, and docs it as removed.

Reply:
I think for 4.14, we should avoid specifics until we are sure of them. But there is some precedent for talking about plans, e.g.:

> Global restricted enforcement for pod security admission is currently planned for the next minor release of OpenShift Container Platform. When this restricted enforcement is enabled, pods with pod security violations will be rejected.

> The Kubernetes community plans to deprecate the use of the Kubernetes controller manager to interact with underlying cloud platforms in favor of using cloud controller managers. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms.

> All other CatalogSource CRs are deprecated and are planned to be removed in a future release.

So I think it would work to say something like this: 
```suggestion
`--included`:: Use this parameter to extract only the manifests that your specific cluster configuration needs. If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` custom resources for any disabled components before installing or updating a cluster. In a future release, this parameter might make the CCO utility (`ccoctl`) `--enable-tech-preview` parameter unnecessary.
```